userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -0,0 +1,161 @@
+
+=== Protocol and Port Setup
+
+Your AWS security group by default provides rules giving access to Kaui and Kill Bill using the HTTP protocol and a specified port number. For HTTPS you will need additional rules with the new protocol and a new port number.
+
+=== Validation
+
+When you create a new certificate you will need to validate your certificate. This identifies the trust hierarchy for the certificate. You may need to setup a temporary CNAME for this purpose, and you may need to provide temporary access via an extra port in your security group.
+
+[[step1]]
+== Step 1: Obtaining a Certificate
+
+There are many ways to obtain a certificate that you can use to protect a particular resource. Here we will describe two that may be of use in particular Kill Bill scenarios running on AWS. These are:
+
+1. Using the AWS Certificate Manager
+2. Using the free CA *Let's Encrypt*
+
+=== Using the AWS Certificate Manager (ACM)
+
+The AWS Certificate Manager (ACM) offers a straightforward way to provide SSL/TLS protection for AWS resources such as load balancers. This is recommended when using the multi-tier or cloud formation installation options. Each of these options makes use of the AWS Elastic Load Balancers (ELBs). For these cases there is no extra cost for using ACM certificates. 
+
+==== 1. Request a Certificate
+
+If you are setting up a new multi-tier or CloudFormation implementation, choose the HTTPS option for your load balancer. You will come to a page that asks for your certificate. click on *Request a new Certificate from ACM*. 
+
+Otherwise, to add HTTPS security to an existing implementation, select *Security, Identity, and Compliance*, then *Certificate Manager* from the Services menu. In this case you will need to edit your load balancer(s) later on.
+
+In either case you will be taken to the main page of the ACM. Initially this will probably show that you have no certificates. On the left menu, click *Request a Certificate*.
+
+The next page will give you the option to request a public or private certificate. The private option may be grayed out. Click *Request a Public Certificate*. The page that appears will ask you to specify several parameters, including one or more domain names and a validation method.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-request-certificate.png[align=center]
+
+==== 2. Setup a CNAME
+
+To validate your certificate, you must first setup a *validation* CNAME. The values to use for the CNAME will be seen shortly.  You will then create the CNAME that you will actually use to access the system.
+
+Enter your domain name, using the wildcard format (e.g., `*.mydomain.com`. In the second pane, select *DNS Validation*. Do not change the other settings. Scroll to the bottom of the page, and click *Request*. The main ACM page will again appear.
+
```

Comment:
Is a screenshot required here?

Reply:
This is now the same page that was seen earlier.

Reply:
okay, it's difficult to visualize what the final pages would look like while reviewing here in Github. I would suggest merging this PR and doing another pass.

Reply:
I agree. Then we can add more screenshots if it seems necessary.